policy/modules/system/modutils.te

```diff
@@ -45,6 +46,8 @@ allow kmod_t self:key write;
 # Read module config and dependency information
 list_dirs_pattern(kmod_t, modules_conf_t, modules_conf_t)
 read_files_pattern(kmod_t, modules_conf_t, modules_conf_t)
+allow kmod_t modules_conf_t:lnk_file read_lnk_file_perms;
```

Comment:
Please use `read_lnk_files_pattern` for uniformity with the previous lines.

Reply:
ok